openapi.yaml

```
@@ -22373,7 +22375,15 @@ components:
           x-linode-cli-display: 1
         kernel:
           type: string
-          description: A Kernel ID to boot a Linode with. Defaults to "linode/latest-64bit".
+          default: linode/latest-64bit
+          description: |
+            A Kernel ID to boot a Linode with. Here are examples of commonly-used kernels:
+
+            * `linode/latest-64bit` (default): Our latest kernel at the time of instance boot/reboot.
+            * `linode/grub2`: The upstream distribution-supplied kernel that is installed on the primary disk, or a custom kernel if installed.
+            * `linode/direct-disk`: The MBR (Master Boot Record) of the primary disk/root device, used instead of a Linux kernel.
+
+            For a complete list of options, use the [Kernels List](/docs/api/linode-instances/#kernels-list) command.
```

```suggestion
            Other than the options above, you can also choose one from the [Kernels List](/docs/api/linode-instances/#kernels-list) API response.
```
I think it would be nicer to word it like this to explicitly say the Kernel List is another option.
Feel free to reject this suggestion or re-word it.

---

`linode/grub2` does appear on the Kernels List response, but on the last page. For example, you should find it on page 4 of the CLI command:

```terminal
$ linode-cli kernels list --page 4
┌──────────────────────────────────┬───────────────────────────┬─────────┬──────────────┐
│ id                               │ label                     │ version │ architecture │
├──────────────────────────────────┼───────────────────────────┼─────────┼──────────────┤
│ linode/3.0.0-x86_64-linode20     │ 3.0.0-x86_64-linode20     │ 3.0.0   │ x86_64       │
│ linode/2.6.39.1-x86_64-linode19  │ 2.6.39.1-x86_64-linode19  │ 2.6.39  │ x86_64       │
│ linode/2.6.39-x86_64-linode18    │ 2.6.39-x86_64-linode18    │ 2.6.39  │ x86_64       │
│ linode/2.6.38-x86_64-linode17    │ 2.6.38-x86_64-linode17    │ 2.6.38  │ x86_64       │
│ linode/2.6.35.4-x86_64-linode16  │ 2.6.35.4-x86_64-linode16  │ 2.6.35  │ x86_64       │
│ linode/2.6.32.12-x86_64-linode15 │ 2.6.32.12-x86_64-linode15 │ 2.6.32  │ x86_64       │
│ linode/2.6.34-x86_64-linode13    │ 2.6.34-x86_64-linode13    │ 2.6.34  │ x86_64       │
│ linode/2.6.34-x86_64-linode14    │ 2.6.34-x86_64-linode14    │ 2.6.34  │ x86_64       │
│ linode/2.6.32.12-x86_64-linode12 │ 2.6.32.12-x86_64-linode12 │ 2.6.32  │ x86_64       │
│ linode/2.6.32-x86_64-linode11    │ 2.6.32-x86_64-linode11    │ 2.6.32  │ x86_64       │
│ linode/2.6.18.8-x86_64-linode10  │ 2.6.18.8-x86_64-linode10  │ 2.6.18  │ x86_64       │
│ linode/2.6.31.5-x86_64-linode9   │ 2.6.31.5-x86_64-linode9   │ 2.6.31  │ x86_64       │
│ linode/2.6.30.5-x86_64-linode8   │ 2.6.30.5-x86_64-linode8   │ 2.6.30  │ x86_64       │
│ linode/2.6.18.8-x86_64-linode7   │ 2.6.18.8-x86_64-linode7   │ 2.6.18  │ x86_64       │
│ linode/2.6.29-x86_64-linode6     │ 2.6.29-x86_64-linode6     │ 2.6.29  │ x86_64       │
│ linode/2.6.28.3-x86_64-linode5   │ 2.6.28.3-x86_64-linode5   │ 2.6.28  │ x86_64       │
│ linode/2.6.28-x86_64-linode4     │ 2.6.28-x86_64-linode4     │ 2.6.28  │ x86_64       │
│ linode/2.6.27.4-x86_64-linode3   │ 2.6.27.4-x86_64-linode3   │ 2.6.27  │ x86_64       │
│ linode/2.6.16.38-x86_64-linode2  │ 2.6.16.38-x86_64-linode2  │ 2.6.16  │ x86_64       │
│ linode/2.6.18.8-x86_64-linode1   │ 2.6.18.8-x86_64-linode1   │ 2.6.18  │ x86_64       │
│ linode/3.5.2-x86_64-linode26     │ 3.5.2-x86_64-linode26     │ 3.5.2   │ x86_64       │
│ linode/grub2                     │ GRUB 2                    │ 2.06    │ x86_64       │
│ linode/direct-disk               │ Direct Disk               │         │ x86_64       │
│ linode/grub-legacy               │ GRUB (Legacy)             │ 2.0.0   │ x86_64       │
│ linode/pv-grub_x86_32            │ pv-grub-x86_32            │ 2.6.26  │ i386         │
│ linode/pv-grub_x86_64            │ pv-grub-x86_64            │ 2.6.26  │ x86_64       │
└──────────────────────────────────┴───────────────────────────┴─────────┴──────────────┘
```

Given this is a pagination issue, I added a note the Kernels List description.